lib/vagrant-service-manager/command.rb

```diff
@@ -41,16 +41,16 @@ def execute
                 # display information about all the providers inside ADB/CDK
                 self.print_all_provider_info
             else
-                self.print_help
+                self.print_help(1)
```

Comment:
if user has entered wrong command which we do not support in sub-commands, it should be non zero exit status.
```
ls --bex ; echo $?
```